syncd/scripts/syncd_init_common.sh

```diff
@@ -49,7 +49,8 @@ case "$(cat /proc/cmdline)" in
     ;;
   *SONIC_BOOT_TYPE=fast*|*fast-reboot*)
     # check that the key exists
-    if [[ $(sonic-db-cli STATE_DB GET "FAST_REBOOT|system") == "1" ]]; then
+    SYSTEM_FAST_REBOOT=`sonic-db-cli STATE_DB GET "FAST_REBOOT|system"`
```

Comment:
Vaibhav and I briefly discussed this change. Since for warm reboot, the name of the table there is `WARM_RESTART_ENABLE_TABLE`, and this key is anyways being changed to use enable/disable instead of 1/not exist, it would be better for consistency purposes to rename the key everywhere to be `FAST_RESTART_ENABLE_TABLE`.